packages/dapp-high-roller/src/components/app-provider/app-provider.tsx

```diff
@@ -78,10 +80,11 @@ export class AppProvider {
     this.cfProvider.on("uninstall", this.onUninstall.bind(this));
     this.cfProvider.on("installVirtual", this.onInstall.bind(this));
 
-    const appId = "0x91907355C59BA005843E791c88aAB80b779446c9";
+    const highRollerAppDefinitionAddr =
+      "0x91907355C59BA005843E791c88aAB80b779446c9";
```

Comment:
I know this was here before but, what is this magic number?

Reply:
no idea